File: static/js/firebase.js
// firebase.js
import { initializeApp, getApps } from "https://www.gstatic.com/firebasejs/9.22.2/firebase-app.js";
import { getDatabase } from "https://www.gstatic.com/firebasejs/9.22.2/firebase-database.js";

// 🔧 Configuração do Banco 1
const configBanco1 = {
  databaseURL: "https://princi-4dfd7-default-rtdb.firebaseio.com/"
};

// 🔧 Configuração do Banco 2
const configBanco2 = {
  databaseURL: "https://sengu-abc16-default-rtdb.firebaseio.com/"
};

// ✅ Inicializa dois apps (usa getApps para evitar duplicações)
const app1 = getApps().find(app => app.name === 'banco1') || initializeApp(configBanco1, 'banco1');
const app2 = getApps().find(app => app.name === 'banco2') || initializeApp(configBanco2, 'banco2');

// 📦 Exporta os dois bancos
const db3 = getDatabase(app1); // dados-teste
const db4 = getDatabase(app2); // testes2

export { db3, db4 };

File: firebase.py
# firebase.py
import firebase_admin
from firebase_admin import credentials, db

def init_firebase(app_name, database_url):
    if not firebase_admin._apps:
        cred = credentials.Certificate("path/to/your/firebase/credentials.json")  # Caminho para o arquivo .json do Firebase
        firebase_admin.initialize_app(cred, {
            'databaseURL': database_url
        }, name=app_name)

def initialize_all_firebase_apps():
    # Inicializar ambos os bancos de dados
    init_firebase("app1", 'https://princi-4dfd7-default-rtdb.firebaseio.com/')
    init_firebase("app2", 'https://sengu-abc16-default-rtdb.firebaseio.com/')
